### PR TITLE
Add distro-specific troubleshooting information

### PR DESCRIPTION
**Pull request by flands**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### Is Fluentd configured properly?
+
+- Is td-agent running? (`systemctl status td-agent`)
+- Check `fluentd.conf` and `conf.d/\*`; ensure `@label @SPLUNK` is added to
+  every source otherwise logs are not collected!
+- Manual configuration may be required to collect logs off the source. Add
+  configuration files to in the `conf.d` directory as needed.
+- Enable debug logging in `fluentd.conf` (`log_level debug`), restart td-agent
+  (`systemctl restart td-agent`), and check `/var/log/td-agent/td-agent.log`
+- While every attempt is made to properly configuration permissions, it is
```
Comment (pmcollins) [inline docs/troubleshooting.md:42]:
```suggestion
- While every attempt is made to properly configure permissions, it is
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  possible td-agent does not have the permission required to collect logs.
+  Debug logging should indicate this issue.
+- This means things are working (requires debug logging enabled): `2021-03-17
+  02:14:44 +0000 [debug]: #0 connect new socket`
+
+### Is OTelCol configured properly?
+
+- Check zpages for samples (`http://localhost:55679/debug/tracez`); may require
```
Comment (pmcollins) [inline docs/troubleshooting.md:50]:
```suggestion
- Check [zpages](https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/zpagesextension) for samples (`http://localhost:55679/debug/tracez`); may require
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  `endpoint` configuration
+- Enable logging exporter and check logs (`journalctl -u
```
Comment (pmcollins) [inline docs/troubleshooting.md:52]:
```suggestion
- Enable [logging exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter) and check logs (`journalctl -u
```

**Review by pmcollins — APPROVED** [review]

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+## Linux Installer
+
+If either the splunk-otel-collector or td-agent services are not properly
+installed and configured:
+
+- Ensure the OS is supported
+- Ensure the OS has systemd installed
+- Ensure pid1 can be accessed (custom configuration required for containerized
+  environments)
+- Check installation logs for more details
```
[the remark this answers is not in the document]
Comment (flands) [reply docs/troubleshooting.md:12]: I am assuming this immediately after running the install script -- so should be on the console.

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Troubleshooting
+
+## Linux Installer
+
+If either the splunk-otel-collector or td-agent services are not properly
+installed and configured:
+
+- Ensure the OS is supported
```
Comment (flands) [inline docs/troubleshooting.md:8]:
```suggestion
- Ensure the OS [is supported](getting-started/linux-installer.md#linux-installer-script)
```

File: docs/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+# Troubleshooting
+
+## Linux Installer
+
+If either the splunk-otel-collector or td-agent services are not properly
+installed and configured:
+
+- Ensure the OS [is supported](getting-started/linux-installer.md#linux-installer-script)
+- Ensure the OS has systemd installed
+- Ensure pid1 can be accessed (custom configuration required for containerized
```
Comment (flands) [inline docs/troubleshooting.md:10]:
```suggestion
- Ensure not running in a containerized environment (for non-production environments see [this post](https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container/) for a workaround)
```

File: docs/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+## Linux Installer
+
+If either the splunk-otel-collector or td-agent services are not properly
+installed and configured:
+
+- Ensure the OS [is supported](getting-started/linux-installer.md#linux-installer-script)
+- Ensure the OS has systemd installed
+- Ensure not running in a containerized environment (for non-production environments see [this post](https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container/) for a workaround)
+  environments)
```
Comment (flands) [inline docs/troubleshooting.md:11]:
```suggestion
```

**Pull request merged**